patches/tools-rust-update_rust.py.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/tools/rust/update_rust.py b/tools/rust/update_rust.py
+index b7e32f0dc764f43535deb4d14f4e1fd2c310e074..d812d7eb5bad62288cd7baf9450fd30f98e146f2 100755
+--- a/tools/rust/update_rust.py
++++ b/tools/rust/update_rust.py
+@@ -35,7 +35,7 @@ sys.path.append(
+ # this back to its previous value _AND_ set `OVERRIDE_CLANG_REVISION` below
+ # to the `CLANG_REVISION` that was in place before the roll.
+ RUST_REVISION = '2e4e2a8f288f642cafcc41fff211955ceddc453d'
+-RUST_SUB_REVISION = 1
++RUST_SUB_REVISION = 3
+ 
+ # If not None, this overrides the `CLANG_REVISION` in
+ # //tools/clang/scripts/update.py in order to download a Rust toolchain that
```

Comment:
...that what?

Reply:
https://github.com/chromium/chromium/blob/8870617af41c11796942d54d6ff70c1fa743bcfb/tools/rust/update_rust.py#L40-L43